pkg/cli/describe.go

```diff
@@ -28,6 +28,7 @@ attribute can be:
 and value, the value of the attribute.
 	`,
 	Run: func(cmd *cobra.Command, args []string) {
+		setUpBuildService()
```

Comment:
First get current phase via control service's Status function. Then connect to the corresponding service.

Reply:
@hemarkus GetProjectNode function belongs in the build service and I can only call it from this service. This is working for all phases, as the functionality is implemented regardless the current state.

What you are mentioning above will be the change once we clean up the control service and create services for each phase.

Reply:
I see ... so we need to address this later.